contents/handbook/people/team-structure/core-analytics.md

```diff
@@ -11,17 +11,25 @@ hideAnchor: true
 
 ## Mission
 
-Make PostHog the best and most performant product analytics platform.
+Enable any user to painlessly, confidently, and consistently surface insights from their data that are useful for their product
+## Goals & Key Results
 
-## Responsibilities
+- Consistency - Maintain performance and availability of performing analysis
+- Confidence - Guarantee data quality 
+- Painlessness - Ensure features provide direct abilities to analyze their data. Users should not have to create unnecessary workarounds
 
-- Make sure data is correct and reliable
-- Make sure queries are quick
-- Increase the types of queries customers can do in PostHog (including frontend changes)
+## Scope
 
-## Customer
+### In Scope
```

Comment:
- Functionality that enables any type of data analysis (e.g. correlation, experimentation).
- Development & maintenance of any feature that supports data analysis (e.g. actions, cohorts).
- Any efforts to ensure data consistency.
- Documentation of all data analysis features.